src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -141,10 +141,13 @@ public static void changeDataTypesInOutputSchema(String jsonFieldList) {
    */
   public static void clickGetSchemaButton() {
     ElementHelper.clickOnElement(CdfPluginPropertiesLocators.getSchemaButton);
-    WaitHelper.waitForElementToBeOptionallyDisplayed(
-      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
-    WaitHelper.waitForElementToBeHidden(
-      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+    WaitHelper.waitForElementToBeNotClickableOptionally(CdfPluginPropertiesLocators.getSchemaButton);
+    // TODO : Remove the commented lines of code once the bug is resolved.
+    // Bug : https://cdap.atlassian.net/browse/PLUGIN-1693
+//    WaitHelper.waitForElementToBeOptionallyDisplayed(
+//      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
+//    WaitHelper.waitForElementToBeHidden(
+//      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
```

Comment:
It looks a bit messy can we keep it this way ?
```
/*
TODO : Remove the commented lines of code once the bug is resolved by Chrome Team.
 Bug : https://cdap.atlassian.net/browse/PLUGIN-1693 
 */
 
 
 /*
 WaitHelper.waitForElementToBeOptionallyDisplayed(
      CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
 WaitHelper.waitForElementToBeHidden(     CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
 */
```

Reply:
Added the multi line comments.

Reply:
LGTM over all, just confused if we should used a single multi line comment or 2 to wrap the bug and code. What do you think ?

Reply:
It's good to wrap and comment in a single line . Will change that and raising one Ex PR .